src/main/java/com/nitnelave/CreeperHeal/block/CreeperBrick.java

```diff
@@ -8,12 +9,12 @@
 class CreeperBrick extends CreeperBlock
 {
 
-    protected CreeperBrick(BlockState blockState)
+    CreeperBrick(BlockState blockState)
```

Comment:
Why do you remove `protected`? This class is supposed to only be constructed by the factory in CreeperBlock.

Reply:
Package-private is more restricted access than protected. Probably should have made a separate commit for restricting member access, I know I did quite a bit of it on non-public things. Sorry about that.